### ISSUE TITLE
Invoice Count grouped by Status from a Single City

### ISSUE THREAD

**Issue by tloriato** [sdec-brasil/dashboard#63]
Both total value as count.

<img width="247" alt="Screen Shot 2019-07-10 at 11 02 35 PM" src="https://user-images.githubusercontent.com/11580564/61016846-80d65580-a367-11e9-8b2b-59c10e87ade0.png">

Related to #60

**Comment by dbeyda** [comment]
This information could be provided by the endpoint that we'll create in #64 .

**Comment by dbeyda** [comment]
##### request
GET `/v1/cities/<city>/status-split?range=1`
##### response
```
{
    "city": {
        "codigoMunicipio": "1100015",
        "cnpj": "18511471000120",
        "municipio.codigoIbge": "1100015",
        "municipio.nome": "Alta Floresta D'Oeste",
        "municipio.uf": "RO",
        "municipio.nomeRegiao": "Leste Rondoniense",
        "municipio.estado.sigla": "RO",
        "municipio.estado.nome": "Rondônia",
        "municipio.regiao.nomeRegiao": "Leste Rondoniense",
        "municipio.regiao.uf": "RO"
    },
    "statusSplit": [
        {
            "estado": 0,
            "count": 9
        },
        {
            "estado": 1,
            "count": 5
        },
        {
            "estado": 2,
            "count": 8
        },
        {
            "estado": 3,
            "count": 12
        },
        {
            "estado": 4,
            "count": 13
        }
    ]
}
```

Range Options:

> 
> /* range options
>   empty - since the beginning of times
>   0 - last 7 days
>   1 - last 30 days
>   2 - last 6 months
>   3 - last year
>   */

**Comment by dbeyda** [comment]
estado and their meaning:

>       // 0 - pendente,
>       // 1 - atrasado,
>       // 2 - pago,
>       // 3 - substituida,
>       // 4 - dados inconsistentes

**Comment by tloriato** [comment]
I don't think this satisfies our use case. And maybe we are going to be needing a new state, while at the same time ignoring the 4th one. 

The 3rd state (substituida) should be ignored as well, as it isn't relevant.

The API has to display the data grouped by 3 status (for each filter obvs.):

- Notes which have been paid (in our database, state 2)
- Notes which have been issued but not paid (in our database, state 0)
- Notes which have been issued and should have been paid (in our database, assuming we keep things well recorded and synchronized, state 1)

However, I feel that it's important (maybe not for this issue) to mark notes in which there's a settlement pending, maybe with a 5th state ("waiting payment")?
